### PR TITLE
POC/learning day: descriptive durations

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves #NUMBER

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/lib/config/services/pidgin.js [hunk 1]
```diff
@@ -115,6 +115,20 @@ const pidgin = {
...
+      },
+      minutes: {
+        singular: 'minuta',
+        plural: 'minut',
+      },
+      hours: {
+        singular: 'godzina',
+        plural: 'godzin',
+      },
+    },
```
Comment (j-pendlebury) [inline src/app/lib/config/services/pidgin.js:131]: this isn't actually pidgin. I used Polish translations because Pidgin doesn't exist on Google Translate

**Comment by j-pendlebury** [comment]
I know this needs loads of unit tests but I ran out of time

**Comment by ghost** [comment]
Storybook error on `StoryPromo`
Screenshot 2019-07-22 at 12 52 25

**Comment by ghost** [comment]
Not all duration translation will match the format `x minutes x seconds` like in Swahili we have `Dakika x Sekunde x`. How do we handle that?

**Comment by j-pendlebury** [comment]
closing in favour of https://github.com/bbc/simorgh/pull/2640

**Pull request closed without merging**